bot/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine as BUILDER
+FROM node:10 as BUILDER
```

Comment:
@gutorc92, why not use the `alpine` image? Also, could you please declare the environment variables `ETCD_URLS` and `TELEGRAM_TOKEN` in this Dockerfile? Thanks!